Godeps/_workspace/src/github.com/GoogleCloudPlatform/kubernetes/pkg/serviceaccount/tokens_controller.go

```diff
@@ -305,6 +306,14 @@ func (e *TokensController) createSecret(serviceAccount *api.ServiceAccount) erro
 	serviceAccount.Secrets = append(serviceAccount.Secrets, api.ObjectReference{Name: secret.Name})
```

Comment:
I was working on something similar (delete and return in case of conflict), with the addition of doing the GET of the service account at the top of the method, and bailing if ResourceVersion didn't match (since that means we'll still get another update notification, and saves us creating and deleting a token)